templates/guides/apim-resources.md

```diff
@@ -4,30 +4,30 @@ page_title: "Gravitee resources"
 
 # Gravitee resources specifics
 
-There are a few things you need to know before you use Gravitee resources
+Gravitee APIM Terraform provider brings a few novelties on how Gravitee resources are managed.
```

Comment:
This guide describes how the Gravitee APIM Terraform provider manages Gravitee resources.